### PR TITLE
wkthmltopdf engine: support for array options

### PR DESCRIPTION
**Pull request by p-try**
--replace can now be used

### CODE REVIEW

File: src/Pdf/Engine/WkHtmlToPdfEngine.php [hunk 1]
```diff
@@ -114,7 +114,11 @@ protected function _getCommand()
 
         $command = $this->_binary;
         foreach ($options as $key => $value) {
-            if (empty($value)) {
+            if (is_array($value)) {
```
Comment (dereuromark) [inline src/Pdf/Engine/WkHtmlToPdfEngine.php]: I would have kept the empty check first, as an empty array also doesnt make much sense right?

Comment (p-try) [inline src/Pdf/Engine/WkHtmlToPdfEngine.php]: You're absolutely right.

**Comment by ADmad** [comment]
Please follow Cake's coding standard https://travis-ci.org/FriendsOfCake/CakePdf/jobs/199559671#L258

Tests needed for the new feature.

**Comment by ADmad** [comment]
@p-try The testsuite should run if you rebase your changes on top of upstream master.

**Comment by ADmad** [comment]
Looks good, can you please squash the commits.

**Pull request merged**